src/scripts/crank.ts

```diff
@@ -279,11 +287,23 @@ async function run() {
 
       }
 
-      await sleep(interval);
+
 
     } catch (e) {
-      log.error(e);
+      if (e instanceof Error) {
+
+        switch (e.message){
```

Comment:
IMO it's better to check error code because the message can change in the future